scripts/extract_translation_templates.py

```diff
@@ -1,15 +1,13 @@
 #!/usr/bin/env python3
-
 import os
 import sys
-import subprocess
 import tempfile
 import logging
 import argparse
 import difflib
-
 import coloredlogs
-
+import requests
+import re
 from eq_translations.entrypoints import handle_extract_template
 
 logger = logging.getLogger(__name__)
```

Comment:
This import block needs re-ordering - stdlib, 3rd party packages, then local modules.

Reply:
Imports are re-ordered now, thanks!